lib/atoms/title1/directive.js

```diff
@@ -0,0 +1,15 @@
+angular
```

Comment:
@andybar2 is it better to use the component name in the directive filename, eg title.directive.js. I see a lot of examples do it that way, and i guess it makes it easier to search for the file in the debugger.